src/index.html

```diff
@@ -7,8 +7,67 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="photo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="item">
+            <a
+              href="#Apple"
+              class="link__item is-active"
+            >APPLE</a>
+            <div class="shape"></div>
+          </li>
+          <li class="item item-2">
```

Comment:
Add blank line between `li` elements
```suggestion
          </li>
          
          <li class="item item-2">
```